.github/pull_request_template.md

```diff
@@ -0,0 +1,58 @@
+```
+INSTRUCTIONS:
+Go through each of these INSTRUCTIONS sections, address them, and then delete them
+```
+
+## Summary
+```
+INSTRUCTIONS:
+Add summary - one or two sentences explaining the purpose of this PR.
+```
+
+## Details
+```
+INSTRUCTIONS:
+Provide further details about the changes, or how they fit into the roadmap.
+You can delete this section if it's not useful.
+```
+
+## Testing
+```
+INSTRUCTIONS:
+Further details about the tests you've added or manually performed -
+you can delete this section if it's not useful.
+```
+
+## Update Instructions
+```
+INSTRUCTIONS:
+If you have any breaking change, or your change enables improvements/updates in clients,
+this is the section to explain that.
+
+In order to allow us to compile aggregated update instructions across PRs, please tag the PR
+with 0+ of the relevant labels:
+* scrypto-lib - Any change to the scrypto library
+* sbor - 
+* manifest - Any change to manifest display, compilation/decompilation
+* transaction - Any change which affects the compiled manifest, or transaction semantics
+* substate - Any change to substates, the state model, or what's stored in the DB
+* native-blueprint-interface - Any change to the interface of native blueprints
+
+If you have a breaking change which doesn't fix into a category above, then tag it with
+the closest label, and discuss in slack/discord.
+```
+
+### User-facing
+```
+INSTRUCTIONS:
+Instructions to a Dapp developer that could be written in eg a Scrypto migration guide
+(eg https://docs-babylon.radixdlt.com/main/scrypto/release_notes/migrating_from_0.7_to_0.8.html )
+which explains how to update their code/manifests for this change.
```

Comment:
I'd suggest we remove this section as well and have dev-ecosystem own the migration guide.